__tests__/app.test.js

```diff
@@ -4,102 +4,146 @@ const app = require('../app/app')
+            })
+    })
+})
 
 describe('GET /api/articles', () => {
     it('should serve an array of all articles without the body property, but with a comment count property', () => {
         return request(app)
-        .get('/api/articles')
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                const articles = response.body.articles.rows
+                expect(articles).toHaveLength(13)
+                articles.forEach((article) => {
+                    expect(article).not.toHaveProperty('body')
+                    expect(article).toHaveProperty('article_id')
+                    expect(article).toHaveProperty('title')
+                    expect(article).toHaveProperty('topic')
+                    expect(article).toHaveProperty('author')
+                    expect(article).toHaveProperty('created_at')
+                    expect(article).toHaveProperty('votes')
+                    expect(article).toHaveProperty('comment_count')
+                })
+            })
+    })
+    it('articles should be sorted by date in descending order', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                const articles = response.body.articles.rows
+                expect(articles).toBeSortedBy('created_at', { descending: false })
+            })
+    })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+    it('should respond with an array of comments for the given article id', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
         .expect(200)
         .then((response)=>{
-            const articles = response.body.articles.rows
-            expect(articles).toHaveLength(13)
-            articles.forEach((article)=>{
-                expect(article).not.toHaveProperty("body")           
-                expect(article).toHaveProperty("article_id")           
-                expect(article).toHaveProperty("title")           
-                expect(article).toHaveProperty("topic")           
-                expect(article).toHaveProperty("author")           
-                expect(article).toHaveProperty("created_at")           
-                expect(article).toHaveProperty("votes")           
-                expect(article).toHaveProperty("comment_count")           
-            })
+            const comments = response.body.comments.rows
+            expect(comments).toHaveLength(11)
+                comments.forEach((comment) => {
+                    expect(comment).toHaveProperty('comment_id')
+                    expect(comment).toHaveProperty('votes')
+                    expect(comment).toHaveProperty('created_at')
+                    expect(comment).toHaveProperty('author')
+                    expect(comment).toHaveProperty('body')
+                    expect(comment).toHaveProperty('article_id', 1)
+                })
```

Comment:
We could add a test to check what happens when the article id is valid but has no comments. As it should be a 200 and sends an empty array 🙂